h/static/scripts/directive/annotation.js

```diff
@@ -655,9 +655,6 @@ function AnnotationController(
   vm.share = function(event) {
     var $container = angular.element(event.currentTarget).parent();
     $container.addClass('open');
-    var shareLinkInput = $container.find('input')[0];
```

Comment:
This logic ensured that the link was selected, ready to copy via pressing Ctrl/Cmd+C as soon as the link appeared. Was it an explicit decision to drop that behavior?